00-course-setup/translations/ja-jp/README.md

```diff
@@ -0,0 +1,140 @@
+
+ご自身のコンピュータの Web ブラウザから直接、Jupyter 環境を使用することもできます。実際クラシックな Jupyter と Jupyter Hub は、自動補完、コード・ハイライトなどの機能を備えた非常に便利な開発環境を提供します。  
+
+Jupyter　をローカルで起動するには、レッスン・コースのディレクトリに移動し、下記のコマンドを実行します：  
+
+```bash
+jupyter notebook
+```
+
+もしくは
+
+```bash
+jupyterhub
+```
+
+次に任意の `.ipynb` ファイルに移動し、ファイルを開いて作業を開始できます。  
+
+### コンテナでの実行
+
+ローカル環境に Python をインストールする代わりに、コンテナ上で実行することもできます。リポジトリ中にコンテナ環境を構築するために必要な設定を含む `.devcontainer` フォルダが存在し、VS Code は起動時にコンテナで実行することを提案します。これは事前に Docker のインストールが必要で、より複雑になるため経験豊富なユーザーに推奨します。
+
+GitHub codespaces を使用する際、API キーを安全に管理する最良の方法は、Codespace Secrets を使用することです。[codespaces のシークレットを管理](https://docs.github.com/en/codespaces/managing-your-codespaces/managing-secrets-for-your-codespaces?WT.mc_id=academic-105485-yoterada)する方法については、このガイドをご参照ください。  
+
+## レッスンと技術要件
+
+The course has 6 concept lessons and 6 coding lessons.
+このコースは、6 つの概念レッスンと 6 つのコーディング・レッスンがあります。
+
+コーディング・レッスンでは、Azure OpenAI サービスを使用します。コードを実行するには、Azure OpenAI サービスへのアクセスと API キーが必要です。[このアプリケーション・フォームを完了](https://go.microsoft.com/fwlink/?linkid=2222006&clcid=0x409?WT.mc_id=academic-105485-yoterada)することで Azure OpenAI へアクセス申請できます。  
+
+アプリケーション・フォームの処理を待つ間、何もできないわけではなく、各レッスン中に、`README.md` ファイルとサンプル・コードを含んでいますので、レッスンの内容をご確認いただけます。
+
+## Azure OpenAI サービスをはじめて使用する場合  
+
+仮に Azure OpenAI サービスをはじめて使用する場合は、[Azure OpenAI サービス・リソースを作成しデプロイする方法](https://learn.microsoft.com/azure/ai-services/openai/how-to/create-resource?pivots=web-portal&WT.mc_id=academic-105485-yoterada)はこちらのガイドをご参照ください。  
+
+## 他の学習者との交流
+
+他の学習者と交流できるように、公式 [Discord AI コミュニティ・サーバー](https://aka.ms/genai-discord?WT.mc_id=academic-105485-yoterada)にチャンネルを作成しました。生成系 AI の技術を向上したいと考える他の方々、たとえば、志が同じ起業家、開発者、学生、そしてどなたとでも交流をお持ちいただく事が可能です。  
+
+[![Discord チャンネルに参加](https://dcbadge.vercel.app/api/server/ByRwuEEgH4)](https://aka.ms/genai-discord?WT.mc_id=academic-105485-yoterada)
+
+このプロジェクトを開発したチームも、この Discord サーバーに参加し学習者を支援します。  
+
+## 貢献
+
+本レッスンはオープンソース・プロジェクトとして公開しています。仮に改善すべき点や問題を発見したら、[プルリクエスト]をお送りいただくか、もしくは、[GitHub の Issue]で報告していただければ幸いです。
+
+このプロジェクトの開発チームは全ての貢献を確認しています。オープンソースにご貢献いただくことは、生成系 AI の分野で新しいキャリアを築くのに良い方法かもしれません。
+
+多くの場合、貢献者はコントリビューター・ライセンス契約（CLA）に同意して頂く必要があります。これは貢献者が持っている権利を、私たちにその貢献を使用する権利を実際に与えて頂くことを宣言する契約です。詳細については[CLA、コントリビューターライセンス契約のウェブサイト](https://cla.microsoft.com?WT.mc_id=academic-105485-yoterada)を訪問してください。
+
+重要なお知らせ：このリポジトリ内に含まれるテキストを翻訳する場合、機械翻訳の使用は避けてください。翻訳の内容はコミュニティで検証するため、ご自身が熟知している言語でのみ翻訳作業にご協力ください。
+
+プル・リクエストを提出すると、CLA-bot が自動的に CLA（コントリビューター・ライセンス契約）の提出が必要かどうかを判断し、プル・リクエストに適切なマーク（例：ラベル、コメント）を付けます。bot の指示にお従いください。この処理は、CLA を使用するすべてのリポジトリで一度だけ行う必要があります。
+
+このプロジェクトは、[Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/?WT.mc_id=academic-105485-yoterada) を採用しています。詳細は、Code of Conduct FAQ をご覧いただくか、追加の質問やコメントがある場合は、[メールで opencode](opencode@microsoft.com) にお問い合わせください。  
+
+## それでは、始めましょう
+
+Now that you have completed the needed steps to complete this course, let's get started by getting an [introduction to Generative AI and LLMs].
+
```

Comment:
```suggestion
```

You've already translated this part into Japanese at line 140, so I think the original text here isn't necessary. 😄